docs/.storybook/preview-head.html

```diff
@@ -1,3 +1,14 @@
+<style>
+  /* Despite this being at the top of the first import in preview.tsx,
+     some component styles are still output before it in the static
+     Storybook build, which causes visual issues due to the `components`
+     layer being applied first.
+
+     Re-declaring the layer order here ensures it happens before any
+     JavaScript-generated styles. */
+  @layer tokens, normalize, reset, components;
```

Comment:
Makes sense. Presumably if we wanted to define any nested layers, this should function fine should still function fine